appserver/tests/functional/embeddedtest/pom.xml

```
@@ -144,6 +109,42 @@
                 <payara.profile>web</payara.profile> 
             </properties>
         </profile>
+        <profile>
+            <id>specificJar</id>
+            <build>
+                <plugins>
+                    <plugin>
+                        <groupId>org.apache.maven.plugins</groupId>
+                        <artifactId>maven-install-plugin</artifactId>
+                        <version>3.1.1</version>
+                        <executions>
+                            <execution>
+                                <id>install-jar-Embedded-${payara.profile}-into-local-maven-repo</id>
+                                <phase>validate</phase>
+                                <configuration>
+                                    <repositoryLayout>default</repositoryLayout>
+                                    <groupId>fish.payara.extras</groupId>
+                                    <artifactId>payara-embedded-${payara.profile}</artifactId>
+                                    <version>${version}</version>
```

We use payara.version everywhere else, I'd like to keep it the same
```suggestion
                                    <version>${payara.version}</version>
```